docs/overview.md

```diff
@@ -16,6 +16,19 @@ JSONata is a lightweight query and transformation language for JSON data. Inspir
 * Install the module from [NPM](https://www.npmjs.com/package/jsonata)
 * Fork the repo on [GitHub](https://github.com/jsonata-js/jsonata)
 
+## Implementations of JSONata
+
+|Language|Link|Notes|Jsonata version|
```

Comment:
```suggestion
## Implementations of JSONata

The following are known implementations of JSONata in addition to the primary implementation in JavaScript in the above repo.

|Language|Link|Notes|JSONata version|
```
Just wanted to add a bit of a clarification about the list, and also `Jsonata` -> `JSONata`